yt_dlp/extractor/telegram.py

```diff
@@ -0,0 +1,42 @@
+from .common import InfoExtractor
+
+
+class TelegramEmbedIE(InfoExtractor):
+    IE_NAME = 'telegram:embed'
+    _VALID_URL = r'https?://t\.me/(?P<channel_name>[^/]+)/(?P<id>\d+)'
+    _TEST = {
+        'url': 'https://t.me/europa_press/613',
+        'info_dict': {
+            'id': '613',
+            'ext': 'mp4',
+            'title': 'Europa Press',
+            'description': '6ce2d7e8d56eda16d80607b23db7b252',
+            'thumbnail': r're:^https?:\/\/cdn.*?telesco\.pe\/file\/\w+',
+        },
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id=video_id)
+
+        webpage_embed = self._download_webpage(f'{url}?embed=1', video_id=video_id)
```

Comment:
The URL passed can be `https://t.me/europa_press/613?embed=1`, `https://t.me/europa_press/613?xxx=yyy`, `https://t.me/europa_press/613#abcd` etc. Use urllib to correctly add the embed query